subiquitycore/models/network.py

```diff
@@ -350,7 +352,14 @@ def get_all_netdevs(self, include_deleted=False):
     def get_netdev_by_name(self, name):
         return self.devices_by_name[name]
 
-    def render(self):
+    def stringify_config(self, config):
+        return '\n'.join([
+            "# This is the network config written by '{}'".format(
```

Comment:
Maybe @vorlonofportland would like a specific breadcrumb here to indicate that this is _the_ source of truth for netplan config; that it is user editable, etc... maybe in the same form as it was written in the previous installer?

Reply:
FTR I'm ok with this as-is, I think breadcrumbs are most important when we need to communicate to a user /not/ to edit a thing and to look elsewhere.